src/package/midlrt/Microsoft.Windows.MidlRT.props

```diff
@@ -0,0 +1,42 @@
+<!--
+***********************************************************************************************
+Copyright (C) Microsoft Corporation.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <PropertyGroup>
+        <!-- Only do this for MSBuild versions below 16.0
+             as it is since done automatically, see https://github.com/microsoft/msbuild/pull/3605-->
+        <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)'  &lt;= '15'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+        
+        <UsingMidlRTPackage>true</UsingMidlRTPackage>
+    </PropertyGroup>
+
+     <PropertyGroup>
+        <PreferredToolArchitecture>x64</PreferredToolArchitecture>
+        <CanReferenceWinRT>true</CanReferenceWinRT>
+        <IsNativeLanguage>true</IsNativeLanguage>
+        <!-- This causes VS to add the platform WinMD references for the target SDK -->
+        <WinMDAssembly>true</WinMDAssembly>
+    </PropertyGroup>
+
+    <ItemDefinitionGroup>
+        <ClCompile>
```

Comment:
remove?

Reply:
remove what? sorry, i can't really tell where your comment is :/

Reply:
oh - remove the ClCompile item config

Reply:
oh yes, gotcha!